docs/src/man/getting_started.md

```diff
@@ -161,10 +161,11 @@ Having seen what some of the rows look like, we can try to summarize the entire
 
 ```jldoctest dataframe
 julia> describe(df)
+2×8 DataFrames.DataFrame
 │ Row │ variable │ mean    │ min │ median  │ max │ nunique │ nmissing │ eltype │
 ├─────┼──────────┼─────────┼─────┼─────────┼─────┼─────────┼──────────┼────────┤
-│ 1   │ A        │ 4.5     │ 1   │ 4.5     │ 8   │ nothing │ nothing  │ Int64  │
-│ 2   │ B        │ nothing │ F   │ nothing │ M   │ 2       │ nothing  │ String │
+│ 1   │ A        │ 4.5     │ 1   │ 4.5     │ 8   │         │          │ Int64  │
+│ 2   │ B        │         │ F   │         │ M   │ 2       │          │ String │
```

Comment:
I'm surprised this column isn't less wide now. I guess you just replaced `nothing` with 7 spaces, right? :-)

Reply:
no i copied and pasted! 

It's a call to `ourstrwidth` that's causing this. `ourstrwidth` prints the value, then uses `take` to get the length. I'll submit a PR to change this behavior so that `ourstrwidth(x::Nothing) = 0`

Reply:
Actually, modifying `ourshowcompact` might be the best thing to do here, because `ourstrwidth` calls `ourshowcompact` when it prints.